locale/ru/index.md

```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Рекомендовано для большинства
   changelog: Список изменений
   api: Документация
-  version-schedule-prompt: Или смотрите на
-  version-schedule-prompt-link-text: график LTS
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```

Comment:
@nodejs/nodejs-ru  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?